var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -117,7 +122,7 @@ def configure_args(self):
         else:
             args.append('--with-kdreg=no')
 
-        for fabric in self.fabrics:
+        for fabric in self.spec.variants['fabrics'].value:
```

Comment:
this is not correct, the idea is to explicitly disable fabrics that were not selected

Reply:
True, whoops. How would you like to handle this? Make all the elements in fabrics conditional? Or handle _ConditionalVariantValues separately in that loop? Make fabrics a list and append when conditions are met instead of using conditional?